app/views/navigation/offices.html

```diff
@@ -117,12 +117,12 @@ <h3 class="table-font-nav">
     <div class="row">
         <div class="col-sm-5 col-md-5">
             <ng-summary title="# {{ 'label.heading.staff' | translate }}">
-                <h1 class="center">07</h1>
+                <h1 class="center">{{staffs.length}}</h1>
```

Comment:
Unnecessarily showing the count twice.... Left block can be used for title and right block can be used to show the number.... if this approach is taken then no need to change the css as well.... same comment applies to centers as well